README.rst

```diff
@@ -43,6 +62,27 @@ How can I run the tests?
    will reduce the danger of race conditions when building containers)
 4. Run ``tox -e $language_stack -- -n auto``
 
+Environment variables
+^^^^^^^^^^^^^^^^^^^^^
+
+You can set the following environment variables to configure the behavior of the BCI-Tests:
+
+.. code-block:: shell-session
+
+    $ export OS_VERSION=15.6                         # Target SLES version
```

Comment:
It would be nice to document all defaults in case nothing is set.

Reply:
Let's address this in another PR.